.changeset/purple-flies-end.md

```diff
@@ -0,0 +1,13 @@
+---
+"@razorpay/blade": minor
```

Comment:
Ideally, this should be treated as a breaking change since users need manual intervention (installing peerDep) to upgrade.

We can make an exception for this PR, to avoid a major release. I'll discuss this with the team.

cc @anurag-rzp @saurabhdaware @kamleshchandnani @chaitanyadeorukhkar

Reply:
We can do it as minor, should be fine

Reply:
minor should be fine I believe since its just additional installation step and shouldn't require consumers to change anything in code as such. And even if it breaks, it breaks on development itself